src/Trixi.jl

```diff
@@ -289,6 +289,10 @@ export PlotData1D, PlotData2D, ScalarPlotData2D, getmesh, adapt_to_mesh_level!,
        adapt_to_mesh_level,
        iplot, iplot!
 
+# Paired Explicit RK (PERK) related exports
+export filter_eig_vals, undo_normalization!, stability_polynomials!,
+       ebisect_stability_polynomial
```

Comment:
```suggestion
export filter_eig_vals, undo_normalization!, stability_polynomials!,
       bisect_stability_polynomial
```

Reply:
Probably these functions do not need to get exported as they will just be called internally by the constructors, right? 
@warisa-r Maybe we remove these exports completely.

Reply:
In my (quite) limited understanding of Julia's package extension mechanisms, extending a function typically requires it to be exported and then imported into the extension file. I have also taken TrixiMakie.jl as an example, in which `iplot` and `iplot!` are exported. Is there a difference between the two cases?

Reply:
I am not quite sure, but I think you don't need to `export` them in order to be able to use them in the extension file. The difference between the Makie extension and the new extension is that `iplot` and `iplot!` are supposed to be used by the user, i.e. we make them part of our API by `export`ing these functions. In contrast, these functions are probably not useful for a user to use directly, i.e. they should not be made available to a user just by `using Trixi` (you can still call them explicitly with, e.g., `Trixi.filter_eig_vals`, but then you know that this is not a public function). Correct me if I'm wrong.

Reply:
Ah! I see the difference now. Thank you for the explanation. I actually have tried this out by something like this:

`
using Trixi: @muladd, filter_eig_vals
function filter_eig_vals()
...
end
`
Unfortunately, this error is produced: `LoadError: error in method definition: function Trixi.filter_eig_vals must be explicitly imported to be extended`. Would it be a good idea to rename the functions in this extension to something like PERK_filtered_eig_vals (maybe not abbreviated) to avoid user confusion?

Reply:
What if you do the following in the extension without exporting `filter_eig_vals` from Trixi.jl?
```julia
using Trixi: filter_eig_vals

function Trixi.filter_eig_vals()
...
end
```

Reply:
Apologies for any confusion. However, I had already commented out `export filter_eig_vals` in `Trixi.jl` when the abovementioned error was produced.

Reply:
But is the error still produced when you append the additional `Trixi.` to the method defintion in the extension?

Reply:
A different error is produced `ERROR: LoadError: UndefVarError: `Trixi` not defined`. I reckon that` filter_eig_vals` can somehow not be loaded properly in the extension using `Trixi.`.

Reply:
Oh, yes, I'm sorry. What if:

```julia
using Trixi: Trixi, filter_eig_vals

function Trixi.filter_eig_vals()
...
end
```

Reply:
Maybe I'm misundersganding something, but you do not want to define new structs or functions in the extension, but only want to add methods, right? If I understand the discussion you linked correctly, they discuss adding new structs or functions in extensions.

Reply:
This works now! Thank you very much!

Reply:
Nice! Thanks for your bearing with me and your contribution overall :)